third-party/build-gcc.sh

```diff
@@ -231,9 +263,15 @@ start_time=$(date +%s)
 fetch_tarballs
 unpack_tarballs
 setup_deps
+
 configure_gcc
 build_gcc
 install_gcc
+
+configure_binutils
+build_binutils
+install_binutils
+
 finalize
 
 cat > $install_dir/bin/enable-gcc.sh <<EOF
```

Comment:
We'd better set LD_LIBRARY_PATH in enable-gcc.sh

Reply:
We better not